.github/workflows/delivery-docker.yml

```diff
@@ -35,27 +35,24 @@ jobs:
             return tag.replace(/^v/, '');
       - name: Checkout workflow
         uses: actions/checkout@v2
-      - name: Checkout source at tag
-        uses: actions/checkout@v2
         with:
-          path: ./src
```

Comment:
I wonder if we'd still like to keep this. Not the local action (`./.github/workflows/actions/pack`), but the ability to update the workflow when re-running a release via `workflow_dispatch`. For example, we can update `.github/workflows/delivery/docker/buildpack.yml` if we need to without needing to bump our release version.

Reply:
We would still need to bump our release version in order to get a change to `buildpack.yml`, though – because it copies from the `src` dir, which is grabbed by the git ref? Or am I misunderstanding this?